### PR TITLE
Adds cloud-provider-openstack acceptance job for manila-provisioner

### PR DESCRIPTION
**Pull request by gman0**
This PR adds an acceptance test for manila-provisioner from kubernetes/cloud-provider-openstack.

The "testing" part of the job is ok, but I bet at least the `roles` section `run.yaml` is incorrect - this is copied from `cloud-provider-openstack-acceptance-test-standalone-cinder`. I'm quite unfamiliar with this CI and I'd be really grateful if somebody could give me an assistance on how to configure the test to run properly for OpenStack Manila with CephFS backend.

### CODE REVIEW

File: playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-k8s
+    - clone-devstack-gate-to-workspace
+    - role: install-devstack
+      environment:
+        OVERRIDE_ENABLED_SERVICES: 'key,c-sch,c-api,c-vol,rabbit,mysql'
```
Comment (kiwik) [inline playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml]: Whether we need to enable manila and ceph in devstack when we run manila-provisioner? if yes, you can use role `create-devstack-local-conf`, like this:

```yaml
  roles:
    - config-golang
    - install-k8s
    - clone-devstack-gate-to-workspace
    - create-devstack-local-conf
      enable_services:
        - 'manila'
        - 'ceph'
    - role: install-devstack
```

Comment (gman0) [reply playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml]: Thanks a lot!

File: playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  become: yes
+  roles:
+    - config-golang
+    - install-k8s
+    - clone-devstack-gate-to-workspace
+    - create-devstack-local-conf
+    - role: install-devstack
+      enable_services:
+        - 'manila'
+        - 'ceph'
```
Comment (kiwik) [inline playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml]: `enable_services` option should be in role `create-devstack-local-conf`, like this:

```yaml
  roles:
    - config-golang
    - install-k8s
    - clone-devstack-gate-to-workspace
    - create-devstack-local-conf
      enable_services:
        - 'manila'
        - 'ceph'
    - role: install-devstack
```

Comment (gman0) [reply playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml]: Ah, thanks!

**Comment by gman0** [comment]
@kiwik thanks a lot for your help!

**Pull request merged**